backend/lrclib.go

```diff
@@ -14,6 +19,38 @@ import (
 	"github.com/dweymouth/supersonic/backend/mediaprovider"
 )
 
+func FetchLrcLibLyricsCached(name, artist, album string, durationSecs int, cacheDir string) (*mediaprovider.Lyrics, error) {
```

Comment:
I'm thinking that this (and the other FetchLrcLib func) should move from static functions to methods on a new LrcLibFetcher struct, that can be initialized in the backend/app.co startup tasks, stored as a new field on the App struct (alongside ServerManager, etc) and then passed to the Now Playing page as just *backend.LrcLibFetcher, to avoid needing to expose the config details directly to the UI. Especially with the follow-up PR adding a custom URL as well.

Reply:
Sure, I'll let you fix up the other 2 comments and then later when I get a bit of time I'll push a commit with the refactoring directly to this branch/PR. Thanks for contributing!

Reply:
Thanks for the help! I just pushed a commit addressing your other two comments. 

Also thanks for the awesome project! I'm really enjoying using it 💯